src/Parser.cpp

```diff
@@ -21,19 +21,19 @@ namespace Parser {
 			return "";
 			break;
 		case MODE:
-			return "<mode>";
+			return "MODE";
 			break;
 		case NUMERIC:
-			return "<value>";
+			return "VALUE";
 			break;
 		case RANGE:
-			return "<min>,<max>";
+			return "RANGE";
```

Comment:
The current style is largely based on git. With this change it might even be prettier but it's unintuitive how RANGE is supposed to be represented. Do you have an examples of other applications that have arguments that look similar to this, especially arguments that need 2 numbers like RANGE?

Reply:
I went looking, but no, `RANGE`-style arguments are not really used. Though, many programs, ones by GNU in particular, use parameter names like `DELIMITER` all over the place.

I think that it's the job of the manual to explain the semantics of `RANGE` and other terms, and `--help` text should not focus on that.

Reply:
Can you give me examples programs use `DELIMITER`? Without looking it up I don't understand what it is or how it relates to `RANGE`.

Why do you think `--help` shouldn't focus on explaining the semantics of arguments? Do you have examples of programs that don't? A counterexample is `ls --help`, which explains the semantics of all the arguments.

Reply:
Nevermind - `DELIMITER` is a simple string. Used by `gawk`, for example. Not related, sorry.

You're right - `ls` does that with `SIZE`. Then it's probably best to do the same - move the syntax description for ranges from the option descriptions, and into a separate block below them.

Reply:
Yea, we could add something like:
```
RANGE is a pair of numbers separated by a comma, where the first number should be less or equal to the second number. It specifies the boundaries of an integer set which will be used in random operations.
```

Reply:
Also since this is now changing let's add a space after the short arguments in the help menu.
```
diff --git a/src/Parser.cpp b/src/Parser.cpp
index 34076f8..c58fc23 100644
--- a/src/Parser.cpp
+++ b/src/Parser.cpp
@@ -53,10 +53,10 @@ namespace Parser {
                case COLOR:
                case TEXT:
                        if (ShortLiteral != "" && LongLiteral != "") {
-                               return "[" + ShortLiteral + valueName + " | " +
+                               return "[" + ShortLiteral + " " + valueName + " | " +
                                       LongLiteral + "=" + valueName + "]";
                        } else if (ShortLiteral != "") {
-                               return "[" + ShortLiteral + valueName + "]";
+                               return "[" + ShortLiteral + " " + valueName + "]";
                        } else {
                                return "[" + LongLiteral + "=" + valueName + "]";
                        }
```